labels/tsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Use same tsconfig than other examples

Reply:
Used the same in https://github.com/CartoDB/deck.gl-examples/blob/master/boundaries/tsconfig.json